sentry_lumberdash/pubspec.yaml

```diff
@@ -15,8 +15,7 @@ environment:
 dependencies:
   lumberdash: ^3.0.0
   meta: ^1.3.0
-  sentry: ^5.0.0
+  sentry: ^6.0.0-0
```

Comment:
This makes it compatible with the current Sentry beta up to but exclusive version 7.x.x.